<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoltageMeasureClass</class>
 <widget class="QMainWindow" name="VoltageMeasureClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VoltageMeasure</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>471</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>生产状态</string>
    </property>
    <widget class="QPushButton" name="BT_State_VolClass">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>电池筛选</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BT_State_IntoCase">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>181</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>电池入支架</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>81</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>生产日志</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_Log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>661</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>741</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 255);</string>
    </property>
    <property name="text">
     <string>电压测试控制系统</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>510</y>
      <width>471</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>系统状态</string>
    </property>
    <widget class="QPushButton" name="pushButton_Stop">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Start">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>启动</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>290</y>
      <width>471</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>生产设备测试</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="BT_PLCTest">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PLC测试</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BT_MySQLTest">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MySQL测试</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BT_HiOKI_Test">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>日置电压仪测试</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_login">
    <property name="title">
     <string>登录</string>
    </property>
    <addaction name="action_operator"/>
    <addaction name="action_technician"/>
    <addaction name="action_engineer"/>
   </widget>
   <widget class="QMenu" name="menu_setting">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_passwordchange"/>
    <addaction name="action_setstandard"/>
   </widget>
   <widget class="QMenu" name="menu_viewdocument">
    <property name="title">
     <string>文件查看</string>
    </property>
    <addaction name="action_viewlog"/>
    <addaction name="action_viewproductdata"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu_login"/>
   <addaction name="menu_setting"/>
   <addaction name="menu_viewdocument"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_operator">
   <property name="text">
    <string>操作员</string>
   </property>
  </action>
  <action name="action_technician">
   <property name="text">
    <string>技术员</string>
   </property>
  </action>
  <action name="action_engineer">
   <property name="text">
    <string>工程师</string>
   </property>
  </action>
  <action name="action_passwordchange">
   <property name="text">
    <string>密码修改</string>
   </property>
  </action>
  <action name="action_setstandard">
   <property name="text">
    <string>设置判定标准</string>
   </property>
  </action>
  <action name="action_viewlog">
   <property name="text">
    <string>日志文件</string>
   </property>
  </action>
  <action name="action_viewproductdata">
   <property name="text">
    <string>生产数据</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="voltagemeasure.qrc"/>
 </resources>
 <connections/>
</ui>
